core/dbt/common/events/types.proto

```diff
@@ -280,6 +280,454 @@ message ProjectCreatedMsg {
     ProjectCreated data = 2;
 }
 
+// E - DB Adapter
+
+// E001
+message AdapterEventDebug {
```

Comment:
why move adapter events into common?

Reply:
I didn't. Don't know why that's showing up as a change. I just merged main into the branch... Git might have gotten confused?

Reply:
It looks like there was a merge error which needed to be resolved but wasn't, and nothing failed because types.proto wasn't rebuilt...